doc/input-output-reference/src/overview/group-variable-refrigerant-flow-equipment.tex

```diff
@@ -639,11 +639,11 @@ \subsubsection{Outputs}\label{outputs-039}
 
 \paragraph{\texorpdfstring{VRF Heat Pump Cooling COP{[]}}{VRF Heat Pump Cooling COP}}\label{vrf-heat-pump-cooling-cop}
 
-This is the operating cooling coefficient of performance (COP) for the heat pump. This value is calculated using the ratio of VRF Heat Pump Total Cooling Rate and Variable Refrigerant Flow Heat Pump Cooling Electric Consumption Rate output variables. Crankcase heater (usually 0 in cooling mode), evaporative condenser water pump, and defrost (usually 0 in cooling mode) consumption rate output variables are included in this calculation. This value is specific to outdoor unit performance in cooling mode, is calculated for each HVAC system time step being simulated, and the results are averaged for the time step being reported.
+This is the operating cooling coefficient of performance (COP) for the heat pump. This value is calculated using the ratio of VRF Heat Pump Total Cooling Rate and VRF Heat Pump Cooling Electricity Rate output variables. Crankcase heater (usually 0 in cooling mode), evaporative condenser water pump, and defrost (usually 0 in cooling mode) consumption rate output variables are included in this calculation. This value is specific to outdoor unit performance in cooling mode, is calculated for each HVAC system time step being simulated, and the results are averaged for the time step being reported.
```

Comment:
Hmm, how does this relate to non-electric systems?

    SetupOutputVariable(state, format("VRF Heat Pump Cooling {} Rate", sFuelType),

    // Calculate OperatingHeatingCOP & OperatingCoolingCOP: VRF Heat Pump Operating COP []
    if (state.dataHVACVarRefFlow->CoolingLoad(VRFCond) && CoolingPLR > 0.0) {
        if (this->ElecCoolingPower != 0.0) {
            // this calc should use delivered capacity, not condenser capacity, use VRF(VRFCond).TUCoolingLoad
            this->OperatingCoolingCOP = (this->TotalCoolingCapacity) /
                                        (this->ElecCoolingPower + this->CrankCaseHeaterPower + this->EvapCondPumpElecPower + this->DefrostPower);
        } else {
            this->OperatingCoolingCOP = 0.0;
        }
    }

Reply:
It seems this is still an open question.  I'll hold from reviewing this for right now.

Reply:
I realize this change only updates the documentation to currently reported text strings. But as I looked at this, this other question popped up. Either use this branch as an update to the correct text string and open another issue to investigate COP with non-electric fuel types, or check into what COP means with non-electric equipment on this branch and update the docs accordingly (or whatever else is needed). I don't use non-electric equipment very often so I don't have a suggestion off the top of my head.

Reply:
Given the timing, I think I will just accept this doc update on its own, and we can investigate COP with non-electric later.